allenact/algorithms/onpolicy_sync/engine.py

```diff
@@ -718,6 +724,9 @@ def collect_step_across_all_task_samplers(
 
         npaused, keep, batch = self.remove_paused(observations)
 
+        if hasattr(self.actor_critic, "sampler_select"):
```

Comment:
Is the idea of this to enable batched evaluation for models using caches?

Reply:
yes!